src/components/Header/Header.js

```diff
@@ -1,9 +1,10 @@
 import React, { Component } from 'react';
 
+import s from './Header.module.css';
```

Comment:
Consider renaming "s" to "styles" to be more descriptive.

Reply:
/fxd